tools/pipelines/templates/include-publish-npm-package-steps.yml

```
@@ -34,29 +31,32 @@ steps:
     targetType: 'inline'
     workingDirectory: $(Pipeline.Workspace)/pack/${{ parameters.artifactPath }}
     ${{ if eq(parameters.namespace, true) }}:
-      ${{ if ne(parameters.feedKind, 'official') }}:
+      ${{ if ne(parameters.feedKind, 'public') }}:
         script: |
-          echo Generating .npmrc for ${{ parameters.feedName }}
-          echo "@fluidframework:registry=${{ parameters.feedName }}" >> ./.npmrc
-          echo "@fluid-example:registry=${{ parameters.feedName }}" >> ./.npmrc
-          echo "@fluid-internal:registry=${{ parameters.devFeedName }}" >> ./.npmrc
-          echo "@fluid-experimental:registry=${{ parameters.feedName }}" >> ./.npmrc
-          echo "@fluid-tools:registry=${{ parameters.feedName }}" >> ./.npmrc
+          echo Generating .npmrc for ${{ parameters.feedUrl }}
+          echo "@fluidframework:registry=${{ parameters.feedUrl }}" >> ./.npmrc
+          echo "@fluid-example:registry=${{ parameters.feedUrl }}" >> ./.npmrc
+          echo "@fluid-internal:registry=${{ parameters.feedUrl }}" >> ./.npmrc
+          echo "@fluid-private:registry=${{ parameters.feedUrl }}" >> ./.npmrc
+          echo "@fluid-experimental:registry=${{ parameters.feedUrl }}" >> ./.npmrc
+          echo "@fluid-tools:registry=${{ parameters.feedUrl }}" >> ./.npmrc
```

Should we just get rid of the scopes here? (in both public and non-public cases). And if we do... maybe just get rid of a lot of the conditionals here. `.npmrc` ends up looking the same in the three cases we're handling here.
```suggestion
          echo "registry=${{ parameters.feedUrl }}" >> ./.npmrc
```

---

good idea

---

Buuuuut... it's a big change, and I don't want to make a change that big to this PR since it's already been validated as
it is. I'll do this separately.